frame/treasury/src/lib.rs

```diff
@@ -243,92 +153,13 @@ pub type ProposalIndex = u32;
 #[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug)]
 pub struct Proposal<AccountId, Balance> {
 	/// The account proposing it.
-	proposer: AccountId,
+	pub proposer: AccountId,
```

Comment:
Why did this need to become pub?

Reply:
Indeed its my bad, no need for pub visibility for this struct members, corrected it.